views/Config.react.js

```diff
@@ -487,15 +489,11 @@ class ConfigPage extends Component {
               title={configs[item]['owner']}
               subheader={updated}
               avatar={
-                configs[item]?.icon?.icon ? (
-                  <img
-                    src={configs[item].icon.icon}
-                    alt=""
-                    className={this.props.classes.configIcon}
-                  />
-                ) : (
-                  this.state.avatar
-                )
+                <ConfigCardAvatar
+                  ownsConfig={ownsConfig}
+                  configOwner={configs[item]?.configOwner}
+                  userAvatar={this.state.avatar}
```

Comment:
if the `configOwner` has the same attributes as a `user` I think we could make this a little simpler and do something like:

```js
<ConfigCardAvatar config={configs[item]} currentUser={user} />
```

and then in the `ConfigCardAvatar` component...

```jsx
const ConfigCardAvatar = (props) => {
  const isCurrentUserConfig = props.user.id === props.config.owner
  const avatarUser = isCurrentUserConfig ? props.user : props.config.configOwner

  return getAvatar({ user: avatarUser })
}
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/196/commits/159c5b43ba6f258f9ca42cfd988c9f29017e7d09